### DISCUSSION TITLE
Hackathon-like Pitch

### DISCUSSION THREAD

**Discussion by 6ar8nas** [vu-vibedosa/apsitvarkom-be#117] [Ideas]
Hey,

As we are nearing to the end of our project, a hackathon-like pitch will be necessary for laboratory work#3. This is why I've created a [Google Document](https://docs.google.com/document/d/1Rp69oOWoyOlYuWv5mHi66Cf7AfS_GJwks3bz2tjPA28/edit?usp=sharing) and [Google Slides](https://docs.google.com/presentation/d/1EXzJx17oJTJZE6C90OavdLguwA1qM1qt8rwjcwxmAiY/edit#slide=id.p) where we could collaborate on this topic.

Feel free to initiate it with what feels most important to you, and as the deadline comes closer, we'll all collectively see what ideas are the most worthwhile for the pitch.

**Comment by BenasB** [comment]
Lecture slides: https://www.canva.com/design/DAFVI0OGYUU/pFuML1KLvMDnby6ekN5NKg/view?utm_content=DAFVI0OGYUU&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
